src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/lowered/gemm_cpu_blocking.hpp

```diff
@@ -18,7 +19,27 @@ class GemmCPUBlocking : public ov::snippets::lowered::pass::BrgemmBlocking<ov::i
 public:
     OPENVINO_RTTI("GemmCPUBlocking", "", BrgemmBlocking)
 
+    /**
+     * @interface DummyPass
+     * @brief The empty pass which is used to force insertion of first specific iteration of loop by K dimension
+     *        Note: the pass is in public section to have opportunity to validate blocking loop in tests
+     * @ingroup snippets
+     */
+    class DummyPass : public snippets::lowered::pass::RangedPass {
+    public:
+        DummyPass() = default;
+        OPENVINO_RTTI("DummyPass", "", snippets::lowered::pass::RangedPass)
+        bool run(snippets::lowered::LinearIR& linear_ir,
+                 snippets::lowered::LinearIR::constExprIt begin,
+                 snippets::lowered::LinearIR::constExprIt end) override;
+        std::shared_ptr<snippets::lowered::pass::PassBase> merge(
+            const std::shared_ptr<snippets::lowered::pass::PassBase>& other) override;
+    };
+
 private:
+    snippets::lowered::SpecificIterationHandlers get_k_loop_handlers(size_t work_amount,
+                                                                     size_t block_size) const override;
+
```

Comment:
Since `GemmCPU` doesn't support blocking loop by K, we don't need all this logic at all (no blocking loop - no handlers are needed). So please revert these changes

Reply:
Done